js_modules/dagit/packages/ui/src/components/Text.tsx

```diff
@@ -7,6 +7,14 @@ interface TextProps {
   color?: string;
 }
 
+export const Title = styled.span<TextProps>`
+  ${({color}) => (color ? `color: ${color};` : null)}
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 29px;
```

Comment:
@braunjj the odd pixel value here is a bit unusual -- should this maybe be 30px or 32px?